test/README.md

```diff
@@ -93,7 +91,7 @@ Complete Tasks 1 *Configure Environment* and 2 *Compile tests*, described in gen
 
   * for a Java9 only features
```

Comment:
This section is blank.  Intentional or WIP?

Reply:
I will remove it for now.